apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -697,6 +697,9 @@ function initializeBlocklyWrapper(blocklyInstance) {
 
     // Hidden workspace where we can put function definitions.
     const hiddenDefinitionWorkspace = new Blockly.Workspace();
+    // The hidden definition workspace is not rendered, so do not try to add
+    // svg frames around the definitions.
+    hiddenDefinitionWorkspace.noFunctionBlockFrame = true;
```

Comment:
this isn't related to this PR, but avoids a null error when we try to create a block frame on a block in the hidden workspace.